lib/Git/Hooks/CheckCommit.pm

```diff
@@ -1,4 +1,4 @@
-use strict;
```

Comment:
Why have you deleted this line? It doesn't seem to be related to the bug you fixed below.

Reply:
Good thing you noticed. I have no idea how that came along.

I reformulated the commit and split it into two.